packages/utils/src/utils/time/timeAgo.ts

```diff
@@ -0,0 +1,26 @@
+const periodValues: Record<string, number> = {
+  year: 31536000,
+  month: 2592000,
+  week: 604800,
+  day: 86400,
+  hour: 3600,
+  min: 60,
+};
+
+export const getTimeAgo = (timeValue: string | number) => {
+  const date = new Date(timeValue);
+  if (Number.isNaN(date.getTime())) {
```

Comment:
Let's call `date.getTime()` only once. I see you call it twice: on the 12th line and the 15th line

Reply:
Done